atom/browser/native_window_mac.mm

```diff
@@ -756,6 +756,17 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
   return bounds;
 }
 
+void NativeWindowMac::SetSize(const gfx::Size& size, bool animate) {
+  gfx::Size minSize = GetMinimumSize();
+  gfx::Size maxSize = GetMaximumSize();
+  // Bound the supplied width and height values with max and min sizes
+  int bWidth = fmax(minSize.width(), size.width());
```

Comment:
There are `SetToMin` and `SetToMax` methods that can do this job:
https://cs.chromium.org/chromium/src/ui/gfx/geometry/size.h?sq=package:chromium&dr=CSs&rcl=1467832600&l=62